packages/dev-frontend/src/components/Stability/Yield.tsx

```diff
@@ -41,6 +41,7 @@ export const Yield: React.FC = () => {
   const remainingLqtyOneYear = remainingStabilityPoolLQTYReward.mul(yearlyHalvingSchedule);
   const remainingLqtyInUSD = remainingLqtyOneYear.mul(lqtyPrice);
   const aprPercentage = remainingLqtyInUSD.div(lusdInStabilityPool).mul(100);
+  const remainingLqtyInUSDTooltip = remainingStabilityPoolLQTYReward.mul(lqtyPrice);
```

Comment:
Hey :) the remaining LQTY is paid out over multiple years, so the stat your showing is not an APR (Annual). Instead you need to multiply by the amount which is distributed over the next year, which is 50% of the remaining rewards.

Reply:
Hi Edward, 
yes I think you are right, but the thing is that the total amount would be multiplied twice in the tooltip (although the final result is correct). If you look at the current numbers: _($532M * 50% / $1.25B) * 100 = 42.65%_  those 532M are already half (_~30.7 * 0.5 * 35_). Besides, _532 * 0.5 / 1250 * 100 = 21,28%_ . 
To clarify: I would only be changing the first number shown in the tooltip, not the result, nor the calculation itself 

![tooltip_APR](https://user-images.githubusercontent.com/78315872/116259400-0e21ae80-a776-11eb-837d-53fb3f6f223f.jpeg)

Reply:
Ah! Gotcha. That makes sense. We should be showing total `$LQTY_REWARDS` remaining, not just this year's. Great spot! Thanks a lot @dvdese 

```suggestion
  const remainingLqtyInUSD = remainingStabilityPoolLQTYReward.mul(lqtyPrice);
```